pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -74,6 +76,12 @@ func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
 	} else if installConfig.BootstrapInPlace.InstallationDisk == "" {
 		errorList = append(errorList, field.Required(field.NewPath("bootstrapInPlace", "installationDisk"),
 			"installationDisk must be set the target disk drive for the installation"))
-	}
+	} else {
+		pathRegex := regexp.MustCompile(`^(/([A-Za-z0-9_-]+)|/)+$`)
```

Comment:
It would be better to instantiate this variable at the top of the file as a `var` similar to the `const` stanza, this would catch any regex at program startup instead of waiting for the codepath to execute. 

Also, since it's going to be top level, it would be ideal to give it an explicit name for it's purpose, something like `installDiskValidationRegex`

Reply:
thks @eggfoobar, this sounds like a good idea, but one question about the catch at program startup, are you referring to move also the `pathRegex.MatchString` outside of the `verifyBootstrapInPlace` function? wouldn't be better to keep it homogeneous?

Reply:
Right, the idea is to make sure you're not caught by surprise if someone was to mistype in that regex and the error would be caught very early on. So putting the ```regexp.MustCompile(`^(/([A-Za-z0-9_-]+)|/)+$`)``` instantiation in a code path where it would always execute helps you identify that, you would keep everything else like `pathRegex.MatchString` in the same place. In regard to keeping things together, since the file is called `bootstrap_in_place.go` I think that's clear on grouping, I would just rename the variable from `pathRegex` to `bootstrapInPlaceDiskRegex` or something similarly explicit.

Reply:
got it, let me push the changes